cerulean.go

```diff
@@ -38,32 +37,11 @@ func New() Cerulean {
 
 	svcs := []services.Service{
 		subscriptionsSVC,
-		resourcegroups.NewService(),
+		resourcegroups.NewService(s),
 	}
 
 	for _, service := range svcs {
-		for endpoint, handlerStruct := range service.GetHandlers() {
-			switch verb := handlerStruct.Verb; verb {
-			case http.MethodGet:
-				e.GET(endpoint, handlerStruct.Func)
-			case http.MethodHead:
-				e.HEAD(endpoint, handlerStruct.Func)
-			case http.MethodPost:
-				e.POST(endpoint, handlerStruct.Func)
-			case http.MethodPut:
-				e.PUT(endpoint, handlerStruct.Func)
-			case http.MethodPatch:
-				e.PATCH(endpoint, handlerStruct.Func)
-			case http.MethodDelete:
-				e.DELETE(endpoint, handlerStruct.Func)
-			case http.MethodConnect:
-				e.CONNECT(endpoint, handlerStruct.Func)
-			case http.MethodOptions:
-				e.OPTIONS(endpoint, handlerStruct.Func)
-			case http.MethodTrace:
-				e.TRACE(endpoint, handlerStruct.Func)
-			}
-		}
+		service.GetAllHandlers(e)
```

Comment:
Any love for `GetHandlers`?

Reply:
Implies the same thing and we don't have any collisions

Reply:
[1:52 PM] Jesse Millar
    All is just the crutch I've fallen back on over the years since a singular s at the end of a function is easy to miss. It's not an ideal solve though.
​[1:52 PM] Ryan Hartje
    Ah, GetHandler vs GetHandlers, ok I see your point
​[1:53 PM] Ryan Hartje
    I think GetServiceHandlers or something is more explicit about what it is